doc/iterations/2023.8/shapes/order_and_filter_from_pages.adoc

```diff
@@ -0,0 +1,38 @@
+= (S) Add an API for filtering and ordering form pages
+
+== Problem
+
+A form can be composed of several pages from various contributions.
+A specifier should be able to order and filter these pages to control the form representation.
```

Comment:
I still believe this should not talk about *specifiers*, but about developers and/or application integrators, who do not define/control the way the various `FormDescriptions` are defined, but want to control how the result appears to their users in their specific application.